k8s/mongodb/mongo-ss.yaml

```diff
@@ -37,14 +37,30 @@ spec:
       terminationGracePeriodSeconds: 10
       containers:
       - name: mongodb
-        image: mongo:3.4.1
+        # TODO(FIXME): Do not use latest in production as it is harder to track
+        # versions during updates and rollbacks. Also, once fixed, change the
```

Comment:
I find this YAML file a bit confusing because there are so many things named `mdb`. (Surely they're not all referring to the same thing?) Could you rename some of them to help with readability and understanding? Maybe `mdb-service`, `mdb-ss`, etc.